tests/cpgrid/addLgrsOnDistributedGrid_test.cpp

```diff
@@ -104,8 +104,8 @@ void refinePatch_and_check(Dune::CpGrid& coarse_grid,
         {
             Dune::cpgrid::Entity<0> entity = Dune::cpgrid::Entity<0>(*data[0], cell, true);
             BOOST_CHECK( entity.hasFather() == false);
-            BOOST_CHECK_THROW(entity.father(), std::logic_error);
-            BOOST_CHECK_THROW(entity.geometryInFather(), std::logic_error);
+            // BOOST_CHECK_THROW(entity.father(), std::logic_error);
+            // BOOST_CHECK_THROW(entity.geometryInFather(), std::logic_error);
```

Comment:
Why is this commented out?

Reply:
Not to see million lines of errors when debugging. I forgot to undo this for the PR, sorry for that!